.golangci.yml

```diff
@@ -13,10 +13,20 @@ run:
 linters:
   disable-all: true
```

Comment:
Why don't we reverse -- enable all linters, but disable ones that we don't want?

Reply:
to avoid surprises, golangci-lint changes its set of linters from version to version.